src/assets/styles/AppTheme.js

```diff
@@ -37,14 +37,24 @@ export const theme = createTheme({
         light: '#000000',
         dark: '#000000',
       },
+      gray: {
+        main: '#616161',
+        light: '#666666',
+        dark: '#9c9c9c',
+      },
+      lightGray: {
+        main: '#a3a3a3',
+        light: '#878787',
+        dark: '#bfbfbf',
+      },
       blue: {
         main: '#4269f5',
         light: '#162f8a',
         dark: '#5e7ae0',
       }
     },
     typography: {
-        color:'#1e1e1e'
+        color:'#000000'
```

Comment:
It's rarely ideal to pure black. A dark grey is best for text. Check with Abena.